src/Exceptions/ApplicationException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Exception;
```

Comment:
špatný namespace a možná by to tu vůbec nemělo být?

Reply:
Nebo by to tu mohlo být, ale v SSH namespace a v kódu db-ex-common to jen chytat a vyhazovat odpovídající lokální user/app exception...

Reply:
Ideálně asi taky tomu dát prefix (SshUserException) aby se to nemuselo aliasovat při používání a nekonfliktilo to s UserException v komponentě...

Reply:
přejmenoval jsem na `Keboola\DbExtractorSSHTunnel\Exception`

Reply:
> Ideálně asi taky tomu dát prefix (SshUserException) aby se to nemuselo aliasovat při používání a nekonfliktilo to s UserException v komponentě...

Takže ti nevadí, že v db-ex-common už UserException a ApplicationException existují? Tj. že budeš muset do catch dát celý název i s namespacem. Nebo si udělat alias v use...  (jestli ne, tak v pohodě a nech to takhle)

Reply:
za chvilku to pošlu do common, udělal jsem to aliasem